components/Fonts.tsx

```diff
@@ -0,0 +1,39 @@
+import { Caveat_Brush as cBrush, Inter, Nunito } from '@next/font/google'
+
+const cbrush = cBrush({
+    weight: '400',
+    variable: '--cbrush-font',
+    preload: true,
+    display: 'swap',
+})
+
+const inter = Inter({
```

Comment:
I don't think we'll need. We'll need Space Mono though, we're using that for rendering code.